visualization/app/codeCharta/ui/fileChooser/fileChooser.component.html

```diff
@@ -7,7 +7,7 @@
 	id="input-file-id"
 	multiple
 	type="file"
-	accept=".json"
+	accept=".json,.gz"
```

Comment:
❤️ 

This should probably be an entry in the changelog under `fixed`.

Reply:
Thanks for the reminder!